src/modules/compliance/tests/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(compliancetests)
 
 include(CTest)
 find_package(GTest REQUIRED)
+FILE(GLOB PROCEDURES procedures/*.cpp)
```

Comment:
Im generally against CMake file GLOBS because CMake cannot track changes and re-run if there are new/removed files but i believe these procedures are generated dynamically? If so, i understand the need to

Reply:
Procedures are not generated dynamically, but at this stage we expect to have a lot of churn both here and in src/modules/compliance/src/lib/procedures, and we want to parallelilze the work on different procedures in different files. Having a glob means we don't need to change CMakeLists.txt with every new file, so we won't have conflicts. In the end we'll just put a static list there. (The same rationale applies to generating ProcedureMap at build  time)

Reply:
i see, can you add a comment above to state that the GLOB is temporary until we stabilize?

Reply:
Done